.github/workflows/options.yml

```diff
@@ -0,0 +1,38 @@
+name: options
+on:
+  push:
+    branches:
+      - main
+  pull_request: null
```

Comment:
Should we change this to run CI on the new option?

Reply:
yes we want to run CI with the new options

Reply:
I suspect it is not running now because the PR is from a fork.

Reply:
It is in fact not entirely clear to me how to resolve this

Reply:
I think you have to enable something : https://github.blog/2020-08-03-github-actions-improvements-for-fork-and-pull-request-workflows/

Reply:
Or maybe even a repo owner @wolfv